Unexpected resolve lock

```
2026-02-10 16:16:55 | [2026/02/10 08:16:55.641 +00:00] [WARN] [subscription_client.go:962] ["subscription client resolve lock fail"] [keyspaceID=1] [regionID=469595] [error="unexpected resolve err: abort:\"Error(Txn(Error(Mvcc(Error(Committed { start_ts: TimeStamp(463852076549016024), commit_ts: TimeStamp(463852076562121557), key: [120, 0, 0, 1, 116, 128, 0, 0, 0, 0, 1, 203, 241, 95, 114, 136, 0, 0, 25, 208, 91, 90, 67] })))))\" , lock: key: 74800000000001CBF15F7288000019D05B5A43, primary: 74800000000001CBF15F698000000000000002013139393430333837FF3031343131333232FF3838300000000000FA, txnStartTS: 463852076549016024, lockForUpdateTS:0, minCommitTs:463852076562121037, ttl: 3003, type: Del, UseAsyncCommit: false, txnSize: 14"] [errorVerbose="unexpected resolve err: abort:\"Error(Txn(Error(Mvcc(Error(Committed { start_ts: TimeStamp(463852076549016024), commit_ts: TimeStamp(463852076562121557), key: [120, 0, 0, 1, 116, 128, 0, 0, 0, 0, 1, 203, 241, 95, 114, 136, 0, 0, 25, 208, 91, 90, 67] })))))\" , lock: key: 74800000000001CBF15F7288000019D05B5A43, primary: 74800000000001CBF15F698000000000000002013139393430333837FF3031343131333232FF3838300000000000FA, txnStartTS: 463852076549016024, lockForUpdateTS:0, minCommitTs:463852076562121037, ttl: 3003, type: Del, UseAsyncCommit: false, txnSize: 14\ngithub.com/tikv/client-go/v2/txnkv/txnlock.(*LockResolver).resolveLock\n\tgithub.com/tikv/client-go/v2@v2.0.8-0.20250917111308-6cbd7e4f9761/txnkv/txnlock/lock_resolver.go:1258\ngithub.com/tikv/client-go/v2/txnkv/txnlock.(*LockResolver).resolveLocks.func2\n\tgithub.com/tikv/client-go/v2@v2.0.8-0.20250917111308-6cbd7e4f9761/txnkv/txnlock/lock_resolver.go:588\ngithub.com/tikv/client-go/v2/txnkv/txnlock.(*LockResolver).resolveLocks\n\tgithub.com/tikv/client-go/v2@v2.0.8-0.20250917111308-6cbd7e4f9761/txnkv/txnlock/lock_resolver.go:596\ngithub.com/tikv/client-go/v2/txnkv/txnlock.(*LockResolver).ResolveLocks\n\tgithub.com/tikv/client-go/v2@v2.0.8-0.20250917111308-6cbd7e4f9761/txnkv/txnlock/lock_resolver.go:431\ngithub.com/pingcap/ticdc/logservice/txnutil.(*resolver).Resolve\n\tgithub.com/pingcap/ticdc/logservice/txnutil/lock_resolver.go:138\ngithub.com/pingcap/ticdc/logservice/logpuller.(*subscriptionClient).handleResolveLockTasks.func2\n\tgithub.com/pingcap/ticdc/logservice/logpuller/subscription_client.go:961\ngithub.com/pingcap/ticdc/logservice/logpuller.(*subscriptionClient).handleResolveLockTasks\n\tgithub.com/pingcap/ticdc/logservice/logpuller/subscription_client.go:979\ngithub.com/pingcap/ticdc/logservice/logpuller.(*subscriptionClient).Run.func7\n\tgithub.com/pingcap/ticdc/logservice/logpuller/subscription_client.go:450\ngolang.org/x/sync/errgroup.(*Group).Go.func1\n\tgolang.org/x/sync@v0.17.0/errgroup/errgroup.go:93\nruntime.goexit\n\truntime/asm_amd64.s:1693\ngithub.com/pingcap/errors.Trace\n\tgithub.com/pingcap/errors@v0.11.5-0.20250523034308-74f78ae071ee/juju_adaptor.go:15\ngithub.com/pingcap/ticdc/logservice/txnutil.(*resolver).Resolve\n\tgithub.com/pingcap/ticdc/logservice/txnutil/lock_resolver.go:140\ngithub.com/pingcap/ticdc/logservice/logpuller.(*subscriptionClient).handleResolveLockTasks.func2\n\tgithub.com/pingcap/ticdc/logservice/logpuller/subscription_client.go:961\ngithub.com/pingcap/ticdc/logservice/logpuller.(*subscriptionClient).handleResolveLockTasks\n\tgithub.com/pingcap/ticdc/logservice/logpuller/subscription_client.go:979\ngithub.com/pingcap/ticdc/logservice/logpuller.(*subscriptionClient).Run.func7\n\tgithub.com/pingcap/ticdc/logservice/logpuller/subscription_client.go:450\ngolang.org/x/sync/errgroup.(*Group).Go.func1\n\tgolang.org/x/sync@v0.17.0/errgroup/errgroup.go:93\nruntime.goexit\n\truntime/asm_amd64.s:1693"]
-- | --
  |   | 2026-02-10 16:16:55 | [2026/02/10 08:16:55.641 +00:00] [ERROR] [lock_resolver.go:1259] ["resolveLock error"] [error="unexpected resolve err: abort:\"Error(Txn(Error(Mvcc(Error(Committed { start_ts: TimeStamp(463852076549016024), commit_ts: TimeStamp(463852076562121557), key: [120, 0, 0, 1, 116, 128, 0, 0, 0, 0, 1, 203, 241, 95, 114, 136, 0, 0, 25, 208, 91, 90, 67] })))))\" , lock: key: 74800000000001CBF15F7288000019D05B5A43, primary: 74800000000001CBF15F698000000000000002013139393430333837FF3031343131333232FF3838300000000000FA, txnStartTS: 463852076549016024, lockForUpdateTS:0, minCommitTs:463852076562121037, ttl: 3003, type: Del, UseAsyncCommit: false, txnSize: 14"] [errorVerbose="unexpected resolve err: abort:\"Error(Txn(Error(Mvcc(Error(Committed { start_ts: TimeStamp(463852076549016024), commit_ts: TimeStamp(463852076562121557), key: [120, 0, 0, 1, 116, 128, 0, 0, 0, 0, 1, 203, 241, 95, 114, 136, 0, 0, 25, 208, 91, 90, 67] })))))\" , lock: key: 74800000000001CBF15F7288000019D05B5A43, primary: 74800000000001CBF15F698000000000000002013139393430333837FF3031343131333232FF3838300000000000FA, txnStartTS: 463852076549016024, lockForUpdateTS:0, minCommitTs:463852076562121037, ttl: 3003, type: Del, UseAsyncCommit: false, txnSize: 14\ngithub.com/tikv/client-go/v2/txnkv/txnlock.(*LockResolver).resolveLock\n\tgithub.com/tikv/client-go/v2@v2.0.8-0.20250917111308-6cbd7e4f9761/txnkv/txnlock/lock_resolver.go:1258\ngithub.com/tikv/client-go/v2/txnkv/txnlock.(*LockResolver).resolveLocks.func2\n\tgithub.com/tikv/client-go/v2@v2.0.8-0.20250917111308-6cbd7e4f9761/txnkv/txnlock/lock_resolver.go:588\ngithub.com/tikv/client-go/v2/txnkv/txnlock.(*LockResolver).resolveLocks\n\tgithub.com/tikv/client-go/v2@v2.0.8-0.20250917111308-6cbd7e4f9761/txnkv/txnlock/lock_resolver.go:596\ngithub.com/tikv/client-go/v2/txnkv/txnlock.(*LockResolver).ResolveLocks\n\tgithub.com/tikv/client-go/v2@v2.0.8-0.20250917111308-6cbd7e4f9761/txnkv/txnlock/lock_resolver.go:431\ngithub.com/pingcap/ticdc/logservice/txnutil.(*resolver).Resolve\n\tgithub.com/pingcap/ticdc/logservice/txnutil/lock_resolver.go:138\ngithub.com/pingcap/ticdc/logservice/logpuller.(*subscriptionClient).handleResolveLockTasks.func2\n\tgithub.com/pingcap/ticdc/logservice/logpuller/subscription_client.go:961\ngithub.com/pingcap/ticdc/logservice/logpuller.(*subscriptionClient).handleResolveLockTasks\n\tgithub.com/pingcap/ticdc/logservice/logpuller/subscription_client.go:979\ngithub.com/pingcap/ticdc/logservice/logpuller.(*subscriptionClient).Run.func7\n\tgithub.com/pingcap/ticdc/logservice/logpuller/subscription_client.go:450\ngolang.org/x/sync/errgroup.(*Group).Go.func1\n\tgolang.org/x/sync@v0.17.0/errgroup/errgroup.go:93\nruntime.goexit\n\truntime/asm_amd64.s:1693"] [debugInfo=]Show context
```

--- a/logservice/logpuller/subscription_client.go
+++ b/logservice/logpuller/subscription_client.go
@@ -967,6 +967,7 @@ func (s *subscriptionClient) handleResolveLockTasks(ctx context.Context) error {
 				zap.Uint64("regionID", regionID),
 				zap.Uint64("targetTs", targetTs),
 				zap.Time("lastRun", lastRun),
+				zap.Any("state", state),
 				zap.Error(err))
 		}
 		resolveLastRun[regionID] = time.Now()

